modular_skyrat/modules/contractor/code/datums/contractor_items.dm

```diff
@@ -149,7 +149,7 @@
 	desc = "Request Syndicate Command to distrupt the station's powernet. Disables power across the station for a short duration."
 	item_icon = "bolt"
 	limited = 2
-	cost = 3
+	cost = 2
```

Comment:
That change is not documented in the PR body, that's a balance change.

Reply:
Forgot that I changed these in the first place, will fix